ui/appui-react/src/appui-react/theme/ThemeManager.tsx

```diff
@@ -13,18 +13,51 @@ import { ThemeProvider } from "@itwin/itwinui-react";
 import type { FrameworkState } from "../redux/FrameworkState";
 import { UiFramework } from "../UiFramework";
 
+/** System preferred color theme.
+ * @public
+ */
+export const SYSTEM_PREFERRED_COLOR_THEME = "SYSTEM_PREFERRED";
+
 /** Enum for the Color Theme string.
  * @public
  */
 export enum ColorTheme {
   Light = "light",
   Dark = "dark",
+  System = SYSTEM_PREFERRED_COLOR_THEME,
+  Inherit = "inherit",
+  HighContrastLight = "high-contrast-light",
+  HighContrastDark = "high-contrast-dark",
 }
 
-/** System preferred color theme.
+/**
+ * List of valid theme values
  * @public
  */
-export const SYSTEM_PREFERRED_COLOR_THEME = "SYSTEM_PREFERRED";
+export type ThemeValues =
+  | keyof {
+      [P in keyof typeof ColorTheme as `${(typeof ColorTheme)[P]}`]: "";
+    }
+  | (string & {});
```

Comment:
```suggestion
export type ThemeType = `${ColorTheme}` | (string & {});
```
We could simplify this by using template literal types.
I like the object intersection approach to keep the auto-completion.
I think we should keep the name singular, maybe rename it to `ThemeType`?

Reply:
Nice, spent a couple hours trying to figure that one and never found this... Definitely what I wanted to do in the first place.

`ThemeType` is the value that iTwinUI-react is using for this, I'd rather not use a clashing name.